web/src/lib/components/album-page/asset-selection.svelte

```diff
@@ -75,6 +75,12 @@
 		</svelte:fragment>
 	</ControlAppBar>
 	<section class="pt-[100px] pl-[70px] grid h-screen bg-immich-bg dark:bg-immich-dark-bg">
-		<AssetGrid isAlbumSelectionMode={true} />
+		<AssetGrid
+			isAlbumSelectionMode={true}
+			options={{
+				isArchived: false,
```

Comment:
I think we should remove `isArchived: false` here since user should be able to add archived photos to the album